.github/commitlint.config.js

```diff
@@ -7,5 +7,9 @@ module.exports = {
     "footer-max-line-length": [0],
     "footer-leading-blank": [0],
     "header-max-length": [0],
+    // Disable some common mistyped scopes and some that should be used
+    "scope-enum": [2, "never", [
+      "kumacp", "kumadp", "kumacni", "kumainit", "*", "madr", "test", "ci", "perf"
```

Comment:
**Question**: These are the disallowed scopes, correct?

I think `ci` is actually useful because there generally isn’t a more specific scope and no scope would be too generic in my opinion. That being said I don’t feel strongly about disallowing it.

Reply:
ci is a type already i think is the idea

Reply:
Yes that's it as I mentioned on slack I'm also happy to move some types to scope.

Reply:
Remember fix, feat etc is supposed to be from the eyes of the user (i.e. for the changelog). So as far as a user is concerned nothing to do with ci is a fix or feat. Something I have to try better to remember myself

Reply:
Ha that's actually a very good point!